tirocks-sys/build.rs

```diff
@@ -31,6 +30,7 @@ fn bindgen_rocksdb(file_path: &Path) {
         .clang_arg("-Irocksdb/include")
         .clang_arg("-Ititan/include")
         .clang_arg("-std=c++11")
+        .clang_arg("-DOPENSSL")
```

Comment:
It's on purpose to always include encryption bindings so that we don't need to keep two binding copies. Since they are not used, so they will not be resolved when encryption is disabled.